logging/bin/deploy_elasticsearch_content_open_openshift.sh

```diff
@@ -0,0 +1,225 @@
+#! /bin/bash
```

Comment:
What is different about this file vs. the non-OpenShift version? Do we need this much duplication?

Reply:
The difference is this one deploys a 3rd log management policy object.  We _could_ make that conditional logic within the original script but we've, so far, kept the OpenShift-specific logic separate.  I lean towards keeping the separation for now but as OpenShift support becomes less experimental, we can look at merging this logic into the original script.